app/src/debug/res/layout/fragment_developer_settings.xml

```diff
@@ -175,6 +175,16 @@
 
         </LinearLayout>
 
+        <Button
+            android:id="@+id/developer_settings_restart_app"
```

Comment:
nit: can you please add `_button` to the end of id? Just easier to work with such ids in future :)